sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -308,7 +308,7 @@ def __init__(
             self._credential = EventhubAzureNamedKeyTokenCredential(credential)  # type: ignore
         else:
             self._credential = credential  # type: ignore
-        self._keep_alive = kwargs.get("keep_alive", 30)
+        self._keep_alive = kwargs.get("keep_alive", 0)
```

Comment:
Pretty sure keep alive was always on by default in uamqp (and therefore EH and SB). Why are we changing this now?

Reply:
It looked like it was disabled for EH, as `None` was getting passed through. It was turned on for SB by default. I was unsure whether it needed to be on the first place but can turn it on